io/read_range.ts

```diff
@@ -97,24 +97,24 @@ export function readRangeSync(
 ): Uint8Array {
   // byte ranges are inclusive, so we have to add one to the end
   let length = range.end - range.start + 1;
-  if (length <= 0) throw new RangeError("Invalid byte range was passed.");
+  if (length <= 0) throw new RangeError("Invalid byte range was passed");
```

Comment:
According to the guide line shouldn't this be something like `Byte range start cannot be larger than end`?